cmd/mt-whisper-importer-reader/main.go

```diff
@@ -196,8 +191,7 @@ func getMetricName(file string) string {
 	splits := strings.Split(file, "/")
```

Comment:
Can't we replace from here until the end with simply:
`return *namePrefix + strings.Replace(strings.TrimSuffix(file, ".wsp"), "/", ".", -1)` ?